### PR TITLE
Add smoke tests for ClickHouse with 20.8 and 21.11.10.1

### PR DESCRIPTION
**Pull request by ebyhr**
https://hub.docker.com/r/yandex/clickhouse-server/tags

### CODE REVIEW

**Comment by ebyhr** [comment]
cc: @tangjiangling

**Comment by tangjiangling** [comment]
@hashhar @ebyhr I am ready to update #10559 and #10594, but both PRs depend on this current PR, can this PR be merged into master now?

File: plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestingClickHouseServer.java [hunk 1]
```diff
@@ -27,13 +28,18 @@
 public class TestingClickHouseServer
         implements Closeable
 {
-    private static final String CLICKHOUSE_IMAGE = "yandex/clickhouse-server:20.8";
+    private static final DockerImageName CLICKHOUSE_IMAGE = DockerImageName.parse("yandex/clickhouse-server");
+    public static final String DEFAULT_VERSION = "20.8"; // 2nd stable version
```
Comment (hashhar) [inline plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestingClickHouseServer.java:33]: The comment will get outdated for sure, let's remove it.

**Review by hashhar — APPROVED** [review]
Thank you

**Pull request merged**